server/parsec/components/postgresql/utils.py

```diff
@@ -133,114 +179,33 @@ def _q_internal_id(**kwargs: Any) -> str:
-    organization_id: str | None = None,
-    table: str = "realm_user_role",
-) -> str:
-    if user is None:
-        assert organization_id is not None and user_id is not None
-        _q_user = q_user_internal_id(
-            organization=organization, organization_id=organization_id, user_id=user_id
-        )
-    else:
-        _q_user = user
-
-    if realm is None:
-        assert organization_id is not None and realm_id is not None
-        _q_realm = q_realm_internal_id(
-            organization=organization, organization_id=organization_id, realm_id=realm_id
-        )
-    else:
-        _q_realm = realm
-
-    return f"""
-COALESCE(
-    (
-        SELECT { table }.role IS NOT NULL
-        FROM { table }
-        WHERE
-            { table }.realm = { _q_realm }
-            AND { table }.user_ = { _q_user }
-        ORDER BY certified_on DESC
-        LIMIT 1
-    ),
-    False
-)
-"""
-
-
-def q_user_can_write_vlob(
-    user: str | None = None,
-    user_id: str | None = None,
-    realm: str | None = None,
-    realm_id: str | None = None,
-    organization: str | None = None,
-    organization_id: str | None = None,
-    table: str = "realm_user_role",
-) -> str:
-    if user is None:
-        assert organization_id is not None and user_id is not None
-        _q_user = q_user_internal_id(
-            organization=organization, organization_id=organization_id, user_id=user_id
-        )
-    else:
-        _q_user = user
-
-    if realm is None:
-        assert organization_id is not None and realm_id is not None
-        _q_realm = q_realm_internal_id(
-            organization=organization, organization_id=organization_id, realm_id=realm_id
-        )
-    else:
-        _q_realm = realm
-
-    return f"""
-COALESCE(
-    (
-        SELECT { table }.role IN ('CONTRIBUTOR', 'MANAGER', 'OWNER')
-        FROM { table }
-        WHERE
-            { table }.realm = { _q_realm }
-            AND { table }.user_ = { _q_user }
-        ORDER BY certified_on DESC
-        LIMIT 1
-    ),
-    False
-)
-"""
+# TODO: This query is unused, still needed?
+# def q_vlob_encryption_revision_internal_id(
+#     encryption_revision: str,
+#     organization_id: str | None = None,
+#     organization: str | None = None,
+#     realm_id: str | None = None,
+#     realm: str | None = None,
+#     table: str = "vlob_encryption_revision",
+# ) -> str:
+#     if realm is None:
+#         assert realm_id is not None
+#         assert organization_id is not None or organization is not None
+#         if organization is None:
+#             _q_realm = q_realm_internal_id(organization_id=organization_id, realm_id=realm_id)
+#         else:
+#             _q_realm = q_realm_internal_id(organization=organization, realm_id=realm_id)
+#     else:
+#         _q_realm = realm
+#     return f"""
+# (
+# SELECT _id
+# FROM {table}
+# WHERE
+#     {table}.realm = {_q_realm}
+#     AND {table}.encryption_revision = {encryption_revision}
+# )
+# """
```

Comment:
Let's just remove anything not needed. We can still browse the `v2` branch when looking for old queries